<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InsertTextUI</class>
 <widget class="QDialog" name="InsertTextUI">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>492</width>
    <height>83</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Ignored">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="6">
    <widget class="QPushButton" name="m_btn_insert">
     <property name="text">
      <string>Insert</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QFontComboBox" name="m_font"/>
   </item>
   <item row="2" column="0" colspan="7">
    <widget class="QTextEdit" name="m_text">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Ignored">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="m_fontsize"/>
   </item>
   <item row="1" column="5">
    <widget class="QToolButton" name="m_textcolor">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="pdfedit.qrc">
       <normaloff>:/images/textpointer.png</normaloff>:/images/textpointer.png</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="m_bold">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="pdfedit.qrc">
       <normaloff>:/images/bold.png</normaloff>:/images/bold.png</iconset>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="m_italic">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="pdfedit.qrc">
       <normaloff>:/images/italic.png</normaloff>:/images/italic.png</iconset>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QPushButton" name="m_underlined">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="pdfedit.qrc">
       <normaloff>:/images/underline.png</normaloff>:/images/underline.png</iconset>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="pdfedit.qrc"/>
 </resources>
 <connections/>
</ui>
